### ISSUE TITLE
373407D6 - Adventure Time: The Secret of Nameless Kingdom

### ISSUE THREAD

**Issue by Sleepy93** [xenia-project/game-compatibility#812]
[Marketplace](https://marketplace.xbox.com/hu-HU/Product/Adventure-Time-Secret-2014/66acd000-77fe-1000-9115-d802373407d6)

Tested on (https://github.com/benvanik/xenia/commit/1eac03a11cca9f19529931bd37f7cfd65afad2c2)

# Issues:
Game is playable with little drawing corruptions.
> libav: overflow in spectral RLE, ignoring

# GPU issues:
> !> 000000A0 GPU: Read from unknown register (044B)
writes to unk. register
- 0081
- 0082
- 0E00
- 0E40
- 1E4E

# Unimplemented:
- XamPartyGetUserList
- XamPartySendGameInvites
- XamPartySetCustomData
- XamPartyGetBandwidth
- XamShowPartyUI
- XamShowCommunitySessionsUI

# Log:
[adventure_time.zip](https://github.com/xenia-project/game-compatibility/files/1253564/adventure_time.zip)

# Labels:
- state-playable
- gpu-corrupt-drawing
- kernel-unimplemented-features

**Comment by Razzile** [comment]
these xam imports look related to xbox live party features so i doubt they'll ever be implemented

**Comment by Sleepy93** [comment]
@Razzile yes. without them game is working so not a problem I just would like to report them too. 👍 
The missing alpha channel for textures is more important :) 
and of course the unkown gpu registers.

**Comment by Sleepy93** [comment]
# Update
looks like nothing changed. 
so just upload the logs.

# Tested 
On (https://github.com/benvanik/xenia/commit/96faf1f8d29d913115b95f8ef5c1c258baf90d21)

# Log file
[at_tsonk.zip](https://github.com/xenia-project/game-compatibility/files/1412310/at_tsonk.zip)


# Labels
- kernel-unimplemented-features
- state-playable
- gpu-corrupt-drawing

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/c3d48eb2fa2e43a19716b52fa5e545f643486b6b

Completed first dungeon. 60-80FPS on 2x resolution and vsync off. No graphical issues.

![at (2)](https://user-images.githubusercontent.com/31154606/100668563-b8a9b880-3321-11eb-97ec-e0834a320c44.png)

<details>
  <summary>More Screenshots</summary>

![at (1)](https://user-images.githubusercontent.com/31154606/100668578-bf383000-3321-11eb-99f9-9acba3df74af.png)
![at (3)](https://user-images.githubusercontent.com/31154606/100668583-c0695d00-3321-11eb-9cfc-b0323b70370e.png)
![at (4)](https://user-images.githubusercontent.com/31154606/100668585-c101f380-3321-11eb-869c-1278787eccf2.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5618836/xenia.zip)

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/166
